.github/workflows/deploy_storybook.yml

```diff
@@ -0,0 +1,38 @@
+name: Test Build and Deploy Storybook to Vercel
+
+on:
+  push:
+    branches:
+      - master
+      - storybook_deploy
+    paths:
+      - packages/components/**
+
+jobs:
+  deploy_storybook:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
```

Comment:
You can probably leave this out, we use matrixes for when we want to define different environments for individual jobs, e.g. a mixture of MacOS builds with Node 12.x.x for job `a` and Windows builds with Node 14.x.x for job `b`.

```suggestion
```

Reply:
I see, because the build fails on Node 15 due to circular dependency errors with our existing manifest, I wanted to keep it at Node 14. Noted on the use case for different OS builds, thanks!